src/user/using-the-aggregate-data-entry-app.md

```diff
@@ -0,0 +1,194 @@
+
+![](resources/images/data_entry_beta/limits-add-button.png) { width=50% }
+
+Once you've added a minimum and maximum limit, click the *Save limits* button.
+
+If a data value already has limits set up, you can change them or remove them. To change the limits, click the *Edit limits* button. To remove the limits, click the *Delete limits* button.
+
+![](resources/images/data_entry_beta/limits-edit-button.png) { width=50% }
+
+Adding, changing, or removing limits requires the correct user privileges.
+
+#### Historical data { #aggregate_data_entry_app.history }
+
+To learn more about a data value, you can see the last twelve values in the *History* section of the data details sidebar. [Make sure the data details sidebar is open](#aggregate_data_entry_app.opening_data_details_sidebar), then open the *History* section
+
+![](resources/images/data_entry_beta/value-history.png) { width=60% }
+
+When working through a data entry form, the *History* section is closed by default to prevent too much network data from being sent for every value.
+
+#### Audit log { #aggregate_data_entry_app.audit_log}
+
+Every data entry cell has an audit log that shows when values were changed and who changed them. To see the audit log, [make sure the data details sidebar is open](#aggregate_data_entry_app.opening_data_details_sidebar), then open the *Audit log* section.
+
+<!-- todo: add image when bug fixes merged -->
+
+When working through a data entry form, the *Audit log* section is closed by default to prevent too much network data from being sent for every value.
+
+
+### Printing { #aggregate_data_entry_app.printing }
+
+To print a data entry form, click the *Options* button in the top bar. From the dropdown menu, you can choose to print the form including cell data values, or an empty form.
+
+![](resources/images/data_entry_beta/print.png) { width=80% }
+
+## Working offline { #aggregate_data_entry_app.working_offline }
+
+You don't need to be connected to the internet or DHIS2 server to enter data. Data entered into forms while you are offline is saved to your local computer. When you reconnect to the internet/server your locally saved data automatically syncs with your DHIS2 server.
+
+To work offline, you need to open the Data Entry (beta) app while you are connected to the internet so the data entry forms can be downloaded and stored on your local computer. Downloading of forms happens automatically in the background.
+
+The badge in the header bar at the top of the screen shows your connection status. If you are not connected to the internet or DHIS2 server, the badge shows *offline*. As well as the offline badge, forms cells you enter data will show the *waiting to sync* status.
+
+![](resources/images/data_entry_beta/offline.png) { width=60% }
+
+Some features aren't available when offline because they need contact with a DHIS2 server to work. Validation, data value history, and data value audit logs aren't available offline.
```

Comment:
change to a note admonition
```suggestion
> **Note**
>
> Some features aren't available when offline because they need contact with a DHIS2 server to work.  
> Validation, data value history, and data value audit logs aren't available offline.
```